zk_toolbox/crates/zk_supervisor/src/messages.rs

```diff
@@ -152,5 +156,18 @@ pub(super) fn msg_running_linter_for_extension_spinner(extension: &Extension) ->
     format!("Running linter for files with extension: .{}", extension)
 }
 
+pub(super) fn msg_running_fmt_for_extension_spinner(extension: Extension) -> String {
```

Comment:
actually, i see this message twice (sometimes), is it ok?
<img width="418" alt="Screenshot 2024-08-19 at 6 54 59 PM" src="https://github.com/user-attachments/assets/31f0584d-3900-4748-87ea-edf72f13727d">